tests/tool/Integration.Tests/E2ETest.cs

```diff
@@ -50,15 +48,15 @@ public async Task UpgradeTest(string scenarioPath, string inputFileName, string
 
             // Copy the scenario files to the temporary directory
             var scenarioDir = Path.Combine(IntegrationTestAssetsPath, scenarioPath);
-            await CopyDirectoryAsync(Path.Combine(scenarioDir, OriginalProjectSubDir), workingDir).ConfigureAwait(false);
+            await FileHelpers.CopyDirectoryAsync(Path.Combine(scenarioDir, OriginalProjectSubDir), workingDir).ConfigureAwait(false);
```

Comment:
this pattern might be cleaner if instead of static methods it's an object that we optional clean up (IDisposable may not work as I like to have the files available on test failures)